hibernate-core/src/main/java/org/hibernate/type/format/jackson/JacksonJsonFormatMapper.java

```diff
@@ -15,11 +15,11 @@
  * @author Christian Beikov
  * @author Yanming Zhou
  */
-public final class JacksonJsonFormatMapper extends AbstractJsonFormatMapper {
+public class JacksonJsonFormatMapper extends AbstractJsonFormatMapper {
```

Comment:
Please copy the contents of this class to `JacksonOsonFormatMapper` and revert the changes in this class.